From 36d52d765ed8be3227b759e20ca2f775c213aef3 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 18 Feb 2013 12:30:48 +0100
Subject: [PATCH] dracut: unset LD_LIBRARY_PATH and GREP_OPTIONS

LD_LIBRARY_PATH and GREP_OPTIONS can influence the image creation,
if set to user defined values, so unset it.

https://bugzilla.redhat.com/show_bug.cgi?id=912299
---
 dracut | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/dracut b/dracut
index 12be718..d2980ef 100755
--- a/dracut
+++ b/dracut
@@ -112,6 +112,8 @@ done
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
 export PATH
+unset LD_LIBRARY_PATH
+unset GREP_OPTIONS
 
 [[ $debug ]] && { 
     export PS4='${BASH_SOURCE}@${LINENO}(${FUNCNAME[0]}): ';
